frontend/src/metabase/parameters/utils/dashboards.ts

```diff
@@ -66,6 +66,26 @@ export function setParameterName(
   };
 }
 
+export function getSourceType(parameter: Parameter): ParameterSourceType {
+  return parameter.source_type ?? null;
+}
+
+export function getSourceOptions(parameter: Parameter): ParameterSourceOptions {
+  return parameter.source_options ?? {};
+}
+
+export function hasValidSourceOptions(
+  sourceType: ParameterSourceType,
+  sourceOptions: ParameterSourceOptions,
+): boolean {
+  switch (sourceType) {
+    case "static-list":
+      return sourceOptions.values != null && sourceOptions.values.length > 0;
+    default:
+      return true;
+  }
+}
```

Comment:
Should we add unit tests for this?

Reply:
I am sometimes on the fence about adding unit tests for utils when it's just newly introduced. Mostly because I'm not sure if the contract is going to stay the same even in the near future, and I feel most of the time they don't since we're not even sure these are valid concepts. But concepts that are batter-tested and has passed some baking time are definitely worth having unit tests.

In this case there's already a unit test for the component (`ParameterSourceSettings`) that utilizes this function and I think that should suffice